docs/2.guide/3.going-further/1.features.md

```diff
@@ -33,6 +33,18 @@ export default defineNuxtConfig({
 })
 ```
 
+### omitLineBreaks
+
+Omits line breaks in `<head>` when rendering HTML.
```

Comment:
Set `omitLineBreaks` to `true` if you prefer to render the head tags without line breaks.

Same with useHead Docs. Is it more appropriate?

Reply:
This paragraph has been removed due to settings tweaks, so I'll mark it as resolved.